### ISSUE TITLE
Mobility

### ISSUE THREAD

**Issue by timsueberkrueb** [qmlrocks/qmlrocks.github.io#1]
Not a priority right now but it would be nice to make the site mobile friendly as some people will like to read it on the go.

**Comment by pierremtb** [comment]
Yeah sure, I'm also willing to make an Android app, could be nice when you
code offline

Le mer. 23 sept. 2015 13:15, Tim Süberkrüb notifications@github.com a
écrit :

**Comment by timsueberkrueb** [comment]
@pierremtb Okay, cool, made with QML I believe :)
btw, can you give me push access to the repo?

**Comment by pierremtb** [comment]
@tim-sueberkrueb Maybe pure android java, so I could learn. Don't know yet.
Go to https://github.com/qmlrocks, you should be invited theorically (I sent it the day before yesterday i think)

**Comment by pierremtb** [comment]
@tim-sueberkrueb For the name I was also thinking about QMLandMe, with a domain like qmland.me

**Comment by timsueberkrueb** [comment]
@pierremtb have you added me as contributor in the settings of the repository?
Good idea, I prefer QMLRocks though :)

**Comment by pierremtb** [comment]
@tim-sueberkrueb Done
